readme.md

```diff
@@ -417,6 +417,7 @@
 - [Programming for Kids](https://github.com/HollyAdele/awesome-programming-for-kids)
 - [Educational Games](https://github.com/yrgo/awesome-eg) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning)
+- [Explorable Explanations](https://github.com/sp4ke/awesome-explorables) - Learn through active reading and reactive documents as defined by Bret Victor. 
```

Comment:
I would drop the `as defined by Bret Victor.` part as not everyone would know how that is, and if they know, then explorables would be obvious to them anyway.